### ISSUE TITLE
feruz teame : Week 3, Be asseccible

### ISSUE THREAD

**Issue by Feruzteame** [HackYourFutureBelgium/class-8#132]
### Prep Work

This part of assignment is planned for thursday. 

### Be assessable Project:
This project is easy but still now i try to understand how aria applied to my code.
 To create html and css is not difficult.
#### Struggle
1. To know what is the difference between aria and accessibility.
2. It is not easy to Know in which part of my code should have to apply aria code.

### Exercises:

In progress.

### Helpfull link 
[Aria] (https://developer.mozilla.org/en-US/docs/Learn/Accessibility/WAI-ARIA_basics)

> [Follow this link](https://github.com/HackYourFutureBelgium/class-6/issues/59) to study a well-done homework issue.

**Comment by MesutBE** [comment]
@Feruzteame I see your one of your struggles maybe I can help you. It is about ARIA.

I visited google home page and inspected it. I choose Search Text Box. There is an ARIA code which I send you a screen capture of it.

![Screenshot from 2020-01-29 15-16-29](https://user-images.githubusercontent.com/59531743/73364192-64215100-42aa-11ea-9f0a-6124d6f6a30f.png)

In the input tag of the code (I highlighted it above) you can see `aria-label="Search`, The web developer says I couldn't place a label on my search bar, so I can place an ARIA code as aria-label="Search, (it is like a embeded label), so the visitor who uses screen reader will know that this box is about "search". And finally this a accessibility of your website.

> **inspected code:** <br>  <input class="gLFyf gsfi" maxlength="2048" name="q" type="text" jsaction="paste:puy29d" aria-autocomplete="both" aria-haspopup="false" autocapitalize="off" autocomplete="off" autocorrect="off" autofocus="" role="combobox" spellcheck="false" title="Search" value="" `aria-label="Search`" data-ved="0ahUKEwiLm424_qjnAhVKbVAKHZP0CfkQ39UDCAQ">

**Comment by Feruzteame** [comment]
Thank  you so much !! very helpful .
